spec/imsc-hrm.html

```diff
@@ -685,8 +692,8 @@ <h2>Paint Text</h2>
       </table>
 
       <p class='note'>The choice of font by the presentation processor can increase rendering complexity. For instance, a cursive
-      font can generally result in a given character yielding different typographical glyphs depending on context, even if latin
-      script is used.</p>
+      font can generally result in a given grapheme yielding different typographical glyphs depending on context,
+      even if latin script is used.</p>
```

Comment:
Here I think using the rigorous term "grapheme" might detract from the note?

I would suggest:

> For instance, a cursive font might select different glyphs for a given character (in order to maintain joining or for the start/end of the word) even in the Latin script.

Reply:
I've been trying to remove references to "character" since it isn't well defined (which is the core point of the issue this PR is trying to resolve!) Hopefully this is still good:

> For instance, a cursive font might select different glyphs for a given code point (in order to maintain joining or for the start/end of the word) even in the Latin script.

?

Reply:
Done in 723cd39.

Reply:
OK, in e8d1cdf I've just changed code point to grapheme, which has a definition link back to the i18n glossary, so I think that's a fair balance between accuracy and comprehensibility, though I agree that grapheme isn't a commonly understood term. Perhaps those readers who don't yet know it will find the experience educational...